packages/react-component-playground/src/components/common.less

```diff
@@ -1,6 +1,17 @@
5Mo9eKrgEamj4uFZkCXMTZ5TfYQkTQ2OlJcqNg0tOY95vSh65LU0XYATbjPRiLGFeAjiYPoArzoazOfwjqYn5u1zfz07CkTjG+bD3oaPaY9HboNImHVhtenblL+2HXheISteBhWXgwlyVGzpCJl2+GsgMh96alDN+vXsMCyLMCj35Yflw7wHakafdf5uiswwVQ67B8AwYcE31qADOFShTkrsIaznYVCONo79XGZugs2FRwgtpXEimd4A0MwPxtOYB6B/wx6FpiAAWF0y3Ik/6SCwIM8X0OXYMDk9G/4K4CdB62/07h6BmFX73a2k705ntI8+wl2Yw6ELNoJ+3nWT9mo3yJGxQFf54RxgvOC/BYAXq7biZkry9H6lD6izu56wM4ZUp0CNZBeNwANehyw3J0DMAEq7CD+hdWGTQgS0FrRFgtOb1gGudeo2YNn6aDtxhiWZjAkNS0LmCmey65czlsxfO5dskn5nRTU+c19hrJoq3NJZEDyNHJMPoNPCmGM76YpEbbAH5Ir9Qik0WmkAgWCgX+javcKFvS/Bm+rEfyS+kqnKf70d6nS9Zp1wF/B4Re8K8XBVP14NGmparUrASk//0RcUpy1UbqBtlWq7aBG0E5fImulwVJTZjFb+Kf2P/7XVPcR7Znq9Dv12rStBA5S33U95oS3vjSi8tsUL/B/5Ix47sSRZIb+zJt5db4WTWTDxpWrtKNrcVpb06h3oJoLlxJZY6Uzdg9RViFQtWR1lwIqvp+upkiIE+OHD6YVhHpjEzQDOBaMbAtIFo+APEzOs4Ngo4sMrgP8JTOClUrLg5bINW10krjDf4T2hGSaWuw+7J2ajEYo1yz5BpzWp9CnTB2bMOavwz5TAcOXRnSVfkUBdmmGdSLPQ3C6Ow0ENl7hpKr42yUhyJ/XTG96Mz/gaOrMA5Hx5y1Ktul2T1rsydmIf5ZQ6X7I1uxX10DMjk69A1AC4nA5eiOpoPvfOhD07/J0WSh1ZI8oTesmd+phiSCHvTQH1NlZ5H3yzUK1BLZ8FX+M0lvXXTUfZdh0W7nBuhCwteErbEpuHIPLpFNLHg2Ymo5YsP6Q9fYiI8PDaNtyG1uuHA6Ubsf2sKtQ8dXLthv/6w57Fpx6Ymna4166TPUd/DVQrPx915oCUzlgkpB89cjjvo4TLQz8lplm7apTkXk02LUv0TcWzOo1tVYW4mxt/frxAubYyPOHmkCv/A+xzzv21TJW1+rQ91luovex+aBG05PX/l0szzY4/qzKEI+p0osiJpv+QlbLrGmL1PWClKbUz3zk1HxOy/rob9ZKpwfRMDszUwXlrLwHEOxsvhjHJj77QMydVEJh2m7hbmDZRdmXYOYEF8WFpqGg7ndkr4Q/kdSg/lICJZaMmGjGBOvhUVJTMJkBpo9VuZkj1RiegwapJFs5ZNzuSLBGDRLLSj832IxnfGBP8JIVWg/SjSRTM43OdYgbb8bkXd2p4J99m4zWcjDkdVIQvQB1CWmzolwEFsRCXFL4XfpRlCr5FeI6fpwRYdwhzNS4fMpop0VJTO6gMli1RCtGQBtinZCr0O6bFlvyKyQq6Do7LBc1tW1k04mqm+J5KmG580gho2qan7j57Re15kIFSEusqaUku6phRG6a0o/bEbMCCTTwi+J24k+jvNgFlUpd80qMJ9uKI7vuXIzgN7PS/WwsJ0TamuaN5nhtFbswQawCmhwMySBoK3aLaey1MgCHvHZcD8dLXkjdazluaDc0ITnFZaQABdZm5Egph+EET3s8Hy9+zzE7Oa68kIk9Wkq0iS7zWrLxa76RJ/FzRobhkY0s6OxWiNZGmUsCpICavcZh2aXOvxlYMnI2DhDmhfi35xGid7pWLEpjQFgyOKhUiO5CJvqVhZTnI1XQDHZMAKGhNQrg5kHlkB/hysICvSsDBCgU2AFGW8O6dvzMCo3qzsSV59yy7QdHUg7qYLNDMPxpzVkT44yPuXR8wYcdFVpos3IDNNlSFFqaWDscI0yZ9pxdphsKcqC/ay2d2mYC803RqqyNWYO+xfUrVfuoePWO9iTeO8iyy9nSvZdDtXLAeWCfSUgAtKCNZHjv9v5wRgHI4lMEbPoCVIC7ljfQ4naIdr4JSmepQJV2+pIU46trssUdnQlbnupbjmlsLM1cSsHqmyW4naGyorTvuzIpQVJ/P+ivekZoleKI/lgUOqxi/F2XQqMsJoY50GHA6if1h2cCV/moJ9JMNw/xOG+2TqNUhGI5iikoagfdyHE/gSFhbDfiZRtnaGmxoSRPYyIx9qQHNtUqO6js6N7cfujR+rnzCdGRyf4/PSBmq8vg81oVabm6S5Xtli2Te+9G2BeY/ZbdJ6nHlDgRXmBLwUA7Mr7F97QazGif9tf9s/+jmliUq5AtRJaIEixnEmMTYiVdzLAhdKQ5eXNmTT5SVw/2cR8Z/Q/akyiTP3g0bg5k58Iq3/464I05aWP0gKPsUomnuVUboz5iMMM7UIu1ohLk6W2ibTV0mKTd5ceoEgzudokNuWx4+UV0HWrhdF+W7Fu5TTG5TVJcrwUzDZSawyODoGmyXDMOphWaGHtS+Dl9DJ4uWDspewsgc7UQQrZ42liYxaZWkOjsiYNULSCA1pAwHQG0IZe5bUJhPQUZ9AF3R6ktVkDDnKFNDzGxZDczKPWXna7G7h1L7pCvgGqjDwp+tCmwRoeoU0p5S+9EsUG+JJaoEnegvQkvgypNkYaMGSQWQT5dYBQpFbKHOftVzALb7unCr1TIUJ9Na1bbdh/E1ol8f7KOs7/dhw9/BpbloMF9t5TBzpPjv2rk7COMan/2aGHNAc95t5yMXGdhj5Dv3VNhhOfjwbt+/eeT1/qlfkxVnXbZ7fvAtOet7HtJohNyEPhBFedwEt+dk7mRejPDwO6ckWTW8PL9+JNpaLU8D+KkxCT/TUDWh7Qy3lYAMG0AWmDiPcncZ57sry0I14yzw+eOFygfZd9wRSS0f4UfAj+/ryrF7dx8zuoScWo0HNJjd
```

Comment:
How about moving the font to `/static` and loading it through `url-loader` (already used for `.png`)?